### DISCUSSION TITLE
Dashboard List-Panel optimisation

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#15669] [Feature Requests]
Currently the list-panel is missing some functionalities that a user would expect after using the relational interfaces. 

The biggest pain point is, that the panel does not uses a pagination when fetching data. This can end up in super slow dashboards when fetching a lot of data.

I think the panel should have comparable options like the relational interfaces:
- use pagination
- sat number of results per page
- add a link to the item / open item in a sidecard

Maybe the panel could also use an abstracted version of the interface in order to have the same options but that probably will bring some difficulties due to the single Query GraphQl implementation of the Dashboards.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
